tastytaps/taps/admin.py

```diff
@@ -1,3 +1,6 @@
 from django.contrib import admin
 
-# Register your models here.
+from .models import Price
+
+
+admin.site.register(Price)
```

Comment:
Do we want the rest here? Or no admin at all? I was thinking we wouldn't use the admin.